docs/CANDIDATE_TECHNOLOGIES.md

```diff
@@ -0,0 +1,159 @@
+# Candidate Technologies for the Gamepad Navigator
+
+## Table of Contents
+
+1. [Introduction](#introduction)
+2. [Factors to be Considered](#factors-to-be-considered)
+3. [Candidate Technologies](#candidate-technologies)
+4. [Analysis](#analysis)
+   - [Custom Browser Extension / Add-on](#custom-browser-extension--add-on)
+   - [Electron.js Desktop App](#electronjs-desktop-app)
+   - [Espruino Pico](#espruino-pico)
+   - [Secondary Extensions / Add-ons](#secondary-extensions--add-ons)
+   - [Bookmarklets](#bookmarklets)
+5. [Conclusion](#conclusion)
+
+## Introduction
+
+This document contains the list of technologies, frameworks, and approaches that can be used to implement **Gamepad Navigator**.
+
+## Factors to be Considered
+
+Each of the available technologies has its advantages and disadvantages, and there's a general tendency to prefer the one with more benefits. But the right approach is to choose the technology according to our requirements and the scope of the project.
+
+So, the factors that should be considered for selecting the candidate technology are:
+
+1. Community's adaptability and familiarity with the technology.
+2. Fulfillment of the project's requirements:
+   - Accept and read inputs from the gamepad.
+   - Relay functional code/script into the browser.
```

Comment:
Not sure what you mean here.  "Send navigational input to a browser"?

Reply:
I meant that we would have to inject or send the functional code into the browser and the website to scan and process inputs within the same. I think you got confused by the order, as this is the first requirement we need to fulfill before the browser can start accepting and reading inputs from the gamepad. Should I reorder it or elaborate it a bit more in the document?

Reply:
I guess I see this as a requirement for working with particular candidate technologies rather than of the project itself.  For example, if we were to go with Espruino, there would be no need to inject anything into the browser.  You should hold onto this as something you look at when digging deeper into your final candidates, but leave the requirements general.   As an example, if you need to inject scripts to read inputs from the gamepad, then IMO it's already covered by that point.

Reply:
I think you are right. I have been thinking from the perspective of the final candidate technology so I wrote it earlier. I'll remove this point.